umm-spec-lib/test/cmr/umm_spec/test/generate_and_parse.clj

```diff
@@ -148,6 +156,15 @@
                                             :Type "UPDATE"}]))
           expected (expected-conversion/convert umm-record metadata-format)
           actual (xml-round-trip :collection metadata-format umm-record)
+         
+          ;; changing everything to set comparison
+          expected (update-in-each expected [:Platforms] update-in-each [:Instruments] update :ComposedOf set)
```

Comment:
@daniel-zamora when you add your changes here, could you please refactor into a function?